core/trino-parser/src/main/java/io/trino/sql/tree/Property.java

```diff
@@ -20,25 +20,41 @@
 import java.util.Optional;
 
 import static com.google.common.base.MoreObjects.toStringHelper;
+import static com.google.common.base.Preconditions.checkState;
 import static java.util.Objects.requireNonNull;
 
 public class Property
         extends Node
 {
     private final Identifier name;
-    private final Expression value;
+    private final Optional<Expression> value;   // empty iff the value is set to DEFAULT
```

Comment:
This is probably ok for now, but eventually it should be modeled with a specific expression representing the `DEFAULT` construct. It will be easier to handle in a more uniform way (in analysis, planning, etc.

Add a TODO.